unit-tests/live/options/test-drops-on-set.py

```diff
@@ -29,44 +29,49 @@ def get_allowed_drops():
     # Our KPI is to prevent sequential frame drops, therefore single frame drop is allowed.
     return 1
 
-def set_new_value(sensor, option, value): 
+def set_new_value(sensor, option, value):
     global after_set_option
     after_set_option = True
     sensor.set_option(option, value)
-    time.sleep( 0.5 )  # collect frames
+    time.sleep(0.5)  # collect frames
```

Comment:
It seem that there are too much unintended formatting changes - please revert the internal spacing (not trailing WS)

Reply:
This has been done by automatic tool formatting the python script. I would like it to remain so that the auto tool can be used later on also.

Reply:
I see.  Regardless, align with @maloel on the formatting settings to avoid back and forth changes